bridge/docs/MP-268:BridgePoolDeployment.puml

```diff
@@ -0,0 +1,21 @@
+@startuml
```

Comment:
what is this?

Reply:
Attempt to document the development in UML. Removed and moved to Wiki